### PR TITLE
ST-1382 Setting up migrations for hearing

### PR DESCRIPTION
**Pull request by osheaf01**
### Change description ###

Setting up migrations for hearing 

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ST-1382

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

**Review by sjamihmcts — DISMISSED** [review]

File: build.gradle [hunk 1]
```diff
@@ -629,7 +629,7 @@ cftlibTest {
   environment 'CCD_DOCUMENT_URL_PATTERN', '.*'
   environment 'DM_STORE_BASE_URL', 'http://localhost:8765'
   environment 'CASE_DOCUMENT_AM_API_ATTACH_DOCUMENT_ENABLED', 'false'
-  environment 'CASE_DATA_STORE_BASEURL', 'http://localhost:4452'
+  environment 'CASE_DATA_STORE_BASEURL', 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
```
Comment (tomxelliott) [inline build.gradle:632]: We probably need to revert this back

File: settings.gradle [hunk 2]
```diff
@@ -13,6 +13,7 @@ rootProject.name = 'sptribs-case-api'
 
 
 //includeBuild '../dtsse-ccd-config-generator'
+//includeBuild '../rse-cft-lib'
```
Comment (tomxelliott) [inline settings.gradle:16]: Do we need to revert this?

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -92,10 +92,14 @@ idam:
...
     url: ${IDAM_API_BASEURL:http://localhost:5000}
+  client:
+    id: ${IDAM_CLIENT_ID:sptribs-case-api}
+    redirect_uri: ${IDAM_CLIENT_REDIRECT_URI:http://localhost:3001/receiver}
+    secret: ${IDAM_CLIENT_SECRET:thUphEveC2Ekuqedaneh4jEcRuba4t2t}
 
   systemupdate:
-    username: ${IDAM_SYSTEM_UPDATE_USERNAME:dummysystemupdate@test.com}
-    password: ${IDAM_SYSTEM_UPDATE_PASSWORD:dummy}
+    username: ${IDAM_SYSTEM_UPDATE_USERNAME:st-test1@mailinator.com}
```
Comment (tomxelliott) [inline src/main/resources/application.yaml:101]: Do wen want to revert this username and password?

**Review by tomxelliott — DISMISSED** [review]
Approved with questions about reverting certain items

File: settings.gradle (hunk 2 above)
Comment (osheaf01) [reply settings.gradle:16]: No, commented out, they're useful when debugging CCD generator and/or CFTLib, but should never be live.

File: build.gradle (hunk 1 above)
Comment (osheaf01) [reply build.gradle:632]: Will do.

Comment (osheaf01) [reply build.gradle]: reverted

File: src/main/resources/application.yaml (hunk 3 above)
Comment (osheaf01) [reply src/main/resources/application.yaml:101]: the dummy entries are garbage, suggest we use the Secrets?

**Review by tomxelliott — APPROVED** [review]
Approved changes - we will need to remediate and rollback some of the changes after testing

**Pull request merged**